_data/JSMarrocco/JSMarrocco.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jean-Simon Marrocco # No longer than 18 characters
+institution: Polytechnique Montreal 🚩 # no longer than 58 characters
+profile_pic: Jean-Simon.jpg # Name and extension of your profile picture(ex. mona.png)
+quote: If debugging is the process of removing software bugs, then programming must be the process of putting them in. - Dijkstra # no longer than 100 characters
```

Comment:
Character count exceeds the limit of 100, you might want to modify this part

Reply:
FIX